website/static/js/profile.js

```diff
@@ -468,19 +479,37 @@ var SocialViewModel = function(urls, modes) {
     var self = this;
     BaseViewModel.call(self, urls, modes);
     TrackedMixin.call(self);
-
+    var i;
     self.addons = ko.observableArray();
 
-    self.personal = extendLink(
-        // Note: Apply extenders in reverse order so that `ensureHttp` is
-        // applied before `url`.
-        ko.observable().extend({
-            trimmed: true,
-            url: true,
-            ensureHttp: true
-        }),
-        self, 'personal'
-    );
+    self.canEditWebsite = ko.observable(false);
+
+    self.editWebsiteButton =  function(){
+        self.canEditWebsite(!self.canEditWebsite());
+    }
+
+    self.profileWebsites = ko.observableArray();
+                
+    self.hasProfileWebsites = ko.computed(function() {
```

Comment:
This, too, becomes unnecessary if blank values are disallowed because you would just be able to check `if (self.profileWebsites())`